### ISSUE TITLE
Use QL for QL to check for omissions

### ISSUE THREAD

**Issue by knewbury01** [github/codeql-coding-standards#522]
as [described here](https://github.com/github/codeql-coding-standards/issues/372#issuecomment-1778870453) there may be a few queries where compiler generated constructs need to be omitted

this issue attempts to capture a more systematic approach to verify if there are remaining opportunities for this

assuming that this issue will have a dependency of [this work](https://github.com/github/codeql-coding-standards/pull/511)

**Comment by zamry09** [comment]
Pada Sab, 3 Feb 2024, 2:16 PG Kristen Newbury ***@***.***>
menulis:

**Comment by rvermeulen** [comment]
Yes, we need that PR to be able to do that. I need to spend some more time optimizing runtime, because it will endlessly rebuild dependency for each QL4QL analysis.
